### ISSUE TITLE
Overwatch 2 error: Game Initialization Failed: I

### ISSUE THREAD

**Issue by DrSay123** [ptitSeb/box64#1148]
Overwatch 2 gives error Game Initialization Failed: I. Tried wine 8.0.2 and 8.13ge, changed box64 build (latest/november), played with Dynarec settings, but still this error appears
[mobox_log.txt](https://github.com/ptitSeb/box64/files/13719371/mobox_log.txt)
![Screenshot_20231219_215510](https://github.com/ptitSeb/box64/assets/87112480/6842d95c-2d21-437a-9ee0-7d2aa5ddcb49)

**Comment by ptitSeb** [comment]
Isn't there some Anti Cheat stuff in that game?

**Comment by DrSay123** [comment]
Well, EasyAntiCheat isnt used there, game works great with wine without box64

**Comment by ptitSeb** [comment]
Ok, I'll check later what's going on.

**Comment by DrSay123** [comment]
So, did you test it?

**Comment by ptitSeb** [comment]
Nope, I didn't even downloaded it yet. quite busy for now, so not sure when I'll be able to test.

**Comment by DrSay123** [comment]
Still didnt test it?

**Comment by hisham2630** [comment]
did you change the mobox dynarec settings like safeflags and strong mem

**Comment by DrSay123** [comment]
Yeah, I did
Still doesnt launch

**Comment by DrSay123** [comment]
Just checked it with wine on Linux pc without box64 - all works
So i assume its box64 bug

**Comment by LukeShortCloud** [comment]
@DrSay123 is this still an issue with the latest Box64?
